AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/resources/ResourceTypesRest.java

```diff
@@ -70,6 +71,20 @@ public List<ResourceTypeDTO> getAllResourceTypes() {
                 .collect(Collectors.toList());
     }
 
+    @Path("/resourceTypes/{name}")
+    @GET
+    @ApiOperation(value = "Get a resource type by name")
+    public ResourceTypeDTO getResourceTypeByName(@PathParam("name") String name) throws NotFoundException, ValidationException {
```

Comment:
Add @NotNull and remove exception handling for empty value.
Use UseCase and throw exception in service.